.github/workflows/tools-api-docs-dev.yml

```diff
@@ -10,7 +10,7 @@ concurrency:
 jobs:
   api-docs:
     name: Build & push Sphinx API docs
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
```

Comment:
Updated upstream in #1762 - fine to leave as is upstream with `-latest` I think.